models/status_test.go

```diff
@@ -21,3 +21,7 @@ import (
 func TestFullStatusSingleBasic(t *testing.T) {
 	CheckJsonFile(t, "status-single-basic.json")
 }
+
+func TestFullStatusProcessIOError(t *testing.T) {
```

Comment:
This only tests that the json is parsed correctly, which is good, but it doesn't test if the right metric names are produced by the exporter. Look at group_processes_test and add the metrics you expect from a given json output.